numba/cuda/cudadrv/driver.py

```diff
@@ -150,6 +155,12 @@ def find_driver():
         _raise_driver_error(errmsg)
 
 
+def find_driver():
+    dlloader, candidates = locate_driver_and_loader()
+    dll, path = load_driver(dlloader, candidates)
```

Comment:
```suggestion
    dll, _ = load_driver(dlloader, candidates)
```
path is unused?

Reply:
It is, but I named it so that you don't have to go and look at `load_driver` to figure out what's being discarded and why.

Reply:
Ah ok, thanks for the explanation, am happy to leave as-is.